.dockerignore

```diff
@@ -0,0 +1,6 @@
+bin
```

Comment:
All these lines refer to files not pushed to the repo. Remember that .dockerignore already ignores the files and folders listed under the .gitignore file.